test/tests/meshgenerators/file_mesh_generator/tests

```diff
@@ -22,6 +22,42 @@
     issues = '#14916'
   [../]
 
+  [./exodus_iga_test]
+    type = 'RunApp'
+    input = '2d_diffusion_iga.i'
+    expect_out = 'Solve Converged'
+    recover = false
+    exodus_version = '>=8.0'
+
+    requirement = 'The system shall have the ability to read IsoGeometric Analysis meshes with rational bases in ExodusII format.'
+    design = 'meshgenerators/FileMeshGenerator.md'
+    issues = '#18768'
+  []
+
+  [./exodus_iga_nosplines_test]
+    type = 'RunApp'
+    input = '2d_diffusion_iga_nosplines.i'
+    expect_out = 'Solve Converged'
+    recover = false
+    exodus_version = '>=8.0'
+
+    requirement = 'The system shall have the ability to remove spline nodes from IsoGeometric Analysis meshes.'
+    design = 'meshgenerators/FileMeshGenerator.md'
+    issues = '#18768'
+  []
+
+  [./exodus_pseudoiga_test]
+    type = 'RunApp'
```

Comment:
Should we do exodiff? I would expect we could un IGA for simple problems such as diffusion and got the correct results. Are we not there yet?

Reply:
We don't have proper output yet.  The CoreForm people have been working on improving libMesh VTK output for IGA meshes, but I don't know if it's there yet or if we can diff it if so, and right now if we output Exodus our RATIONAL_BERNSTEIN elements will be treated like LAGRANGE and the geometry distorted.  I don't want to put in a test employing that.

Reply:
It's an interesting question, whether we should disable imperfect output types of IGA meshes entirely in the meantime.  Exodus IGA output is still useful (sometimes as a "close enough in the eyeball norm" way to visualize things, sometimes as a "look at the control points for their own sake" tool) though.

Reply:
I think doing VTK output with VTKDiff would work, as far as I know it is all working as expected.

Reply:
CI agrees, when we have VTK 9; I'll just need to disable the tests for VTK 8 or lower.